tests/resources/base_cpu/demo.sh

```diff
@@ -2,6 +2,9 @@
 # Copyright (C) 2019-2020 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
 
+apt update
+apt install -y build-essential curl cmake
```

Comment:
maybe add curl, cmake, gcc, g++ to base and custom Dockerfiles. And install only make in the test.

Reply:
OV installs the same packages in test scripts
https://github.com/openvinotoolkit/openvino/tree/master/scripts/demo